src/extension/background.js

```diff
@@ -51,11 +56,20 @@ const messageHandlers = {
   setTheme: message => {
     const theme = normalizeTheme(message.theme);
     log("setTheme", theme);
-    if (!message.isPreview) {
-     storeTheme({ theme });
-    }
+    storeTheme({ theme });
     applyTheme({ theme });
   },
+  previewTheme: message => {
+    if (message.theme) {
+      const theme = normalizeTheme(message.theme);
+      log("previewTheme", theme);
+      applyTheme({ theme });
+      isThemePreview = true;
+    } else {
+      fetchTheme().then(applyTheme);
```

Comment:
Should there be a `log` here? E.g. `log("revertTheme");`

Reply:
Done. A few notes to consider: 
- The onMouseLeave event sends an empty object via `postMessage`, effectively telling the extension to revert to the stored theme. When this occurs the `theme` argument passed to `log` will be `undefined`, such that the console log will be:
`[FirefoxColor background] previewTheme undefined`. 
- In the `setTheme` message listener, perhaps we may wish to also destructure `theme` from `messages` for the sake of consistency.